From 772fade0232b98460ba9557eba703593226ce804 Mon Sep 17 00:00:00 2001
From: Samuel Dionne-Riel <samuel@dionne-riel.com>
Date: Fri, 23 Oct 2020 20:59:10 -0400
Subject: [PATCH] Work around missing -pthread

It is unknown why `pkg-config` does not return `-pthread` for zeromq.

---
 mrbgem.rake | 1 +
 1 file changed, 1 insertion(+)

diff --git a/mrbgem.rake b/mrbgem.rake
index ff42825..3b7ebdc 100644
--- a/mrbgem.rake
+++ b/mrbgem.rake
@@ -75,4 +75,5 @@ MRuby::Gem::Specification.new('mruby-zmq') do |spec|
       spec.cc.defines << 'ZMQ_BUILD_DRAFT_API=1'
     end
   end
+  spec.linker.flags_before_libraries << "-pthread"
 end
-- 
2.28.0

